src/test/java/uk/ac/ebi/subs/validator/taxon/TaxonomyValidatorTest.java

```diff
@@ -0,0 +1,72 @@
+package uk.ac.ebi.subs.validator.taxon;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.client.HttpClientErrorException;
+import uk.ac.ebi.subs.data.submittable.Sample;
+import uk.ac.ebi.subs.validator.data.SingleValidationResult;
+import uk.ac.ebi.subs.validator.taxon.core.Taxonomy;
+import uk.ac.ebi.subs.validator.taxon.core.TaxonomyService;
+import uk.ac.ebi.subs.validator.taxon.core.TaxonomyValidator;
+
+import java.util.UUID;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class TaxonomyValidatorTest {
+
+    private final String SUCCESS_MESSAGE = "Valid taxonomy";
```

Comment:
If you changed the messages in the service, then change it here as well, please.